docker-server.Dockerfile

```diff
@@ -2,52 +2,25 @@
 # See the DOCKER.md file in the root of the project for more information.
 # Please run `docker compose up` from the root of the project to run the docker environment for
 # the UCW-APP project, which this Dockerfile is part of.
-ARG WRKDR=/opt/app
-
 FROM alpine:3.20.3 AS base
 ENV NODE_VERSION 20.15.0
 
+WORKDIR /usr/src/app
+
 RUN apk --update --no-cache --virtual add nodejs npm \
     && rm -rf /var/cache/apk/*
 
-FROM base AS pruner
-RUN npm i -g turbo
-ARG APP
-ARG WRKDR
-
-WORKDIR ${WRKDR}
-
-COPY . ${WRKDR}
-RUN turbo prune --scope=${APP} --docker
-
-FROM base AS builder
-ARG APP
-ARG WRKDR
-
-WORKDIR ${WRKDR}
+# Copy app source
+COPY . .
+ 
+RUN npm ci
```

Comment:
if you do `nom ci --omit=dev` it could trim the size down as well, since we shouldn't need the devDeps for prod.